cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -107,7 +111,8 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("single()", "functions-single") {
-      p("`single()` returns true if the predicate holds for exactly one of the elements in the given list.")
+      p("`single()` returns true if the predicate holds for exactly one of the elements in the given list." +
+        " Returns `null` if the list is `null` or all of its elements are `null`.")
```

Comment:
"Returns `null` if the..." -> "`null` is returned if the ..."  (to make it a sentence, rather than a clause)